### ISSUE TITLE
When window size > 1600, things start to get funky

### ISSUE THREAD

**Issue by akariv** [openspending/os-style-guide#5]

**Comment by smth** [comment]
Funky in a good way, right?

**Comment by pwalsh** [comment]
https://www.youtube.com/watch?v=8rrOdcnFbAY

**Comment by akariv** [comment]
Is there a bad way, @smth ?

What I've noticed is mainly the top menu disappearing.
In the current packager (which is about to change), the standard OKI footer is positioned incorrectly.

Adam

**Comment by smth** [comment]
Yeah. This style guide page was based on a preexisting template, which I think assumed that once you had two columns, the page would no longer need to scroll, and so you wouldn't need the menu. Not ideal, but this should only effect the style guide page, so probably not worth worrying about.

I'm not attempting to provide layout here, the footer issue should probably be raised elsewhere.
